### PR TITLE
[15.0] [MIG] Migrated crm_operating_unit

### PR DESCRIPTION
**Pull request by Rajan-SCS**

### CODE REVIEW

**Comment by AaronHForgeFlow** [comment]
Hi @Rajan-SCS  thanks for the PR. Can you check why the commit messages has been renamed? All of them has the header [PATCH X/25] that is not the actual original commit message

**Comment by Rajan-SCS** [comment]
@AaronHForgeFlow could you please check the PR

File: crm_operating_unit/tests/test_crm_operating_unit.py [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+                "company_ids": [(4, company.id)],
+                "operating_unit_ids": [(4, ou.id) for ou in operating_units],
+                "groups_id": [(6, 0, group_ids)],
+            }
+        )
+        return user
+
+    def _create_crm_team(self, uid, operating_unit):
+        """Create a sale order."""
+        crm = self.crm_team_model.with_context(
```
Comment (nikul-serpentcs) [inline crm_operating_unit/tests/test_crm_operating_unit.py:57]:
```suggestion
        crm = self.crm_team_model.with_context(
            mail_create_nosubscribe=True, mail_create_nolog=True).create({
                "name": "CRM team", "operating_unit_id": operating_unit.id,
                "user_id": uid})
```

**Comment by nikul-serpentcs** [comment]
@AaronHForgeFlow now ready to review

**Review by AaronHForgeFlow — APPROVED** [review]
Code + Functional ok!

**Review by JuanyDForgeflow — APPROVED** [review]
Functional review is OK

**Comment by AaronHForgeFlow** [comment]
Hi @Rajan-SCS , can you check if running pre-commit again is needed here?

**Comment by JuanyDForgeflow** [comment]
@Rajan-SCS please rebase and run pre-commit again

**Review by BT-apadron — APPROVED** [review]
LGTM

**Comment by AaronHForgeFlow** [comment]
Let's see if pre-commit passes now

/ocabot rebase

**Comment by AaronHForgeFlow** [comment]
Same here @Rajan-SCS pre-commit is failing, just let us know if you have time to fix it.

**Comment by AaronHForgeFlow** [comment]
Superseded here: https://github.com/OCA/operating-unit/pull/528

File: crm_operating_unit/tests/test_crm_operating_unit.py (hunk 1 above)
Comment (AaronHForgeFlow) [reply crm_operating_unit/tests/test_crm_operating_unit.py:57]: Comment attended in the supersede

**Pull request closed without merging**